tests/test_lag_2.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import time
+import logging
+from ptf_runner import ptf_runner
+from platform_fixtures import conn_graph_facts
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost, conn_graph_facts):
+    # TODO: get the testbed_type
+
+    lag_facts = duthost.lag_facts(host = duthost.hostname)['ansible_facts']
+    if lag_facts['names'] == []:
+        pytest.skip("No lag configuration found in %s" % duthost.hostname)
+
+    test_minlink = True
+    test_rate = True
+
+    # Add/update the public key
+    # TODO: convert add_container_to_inventory.yml
```

Comment:
Not found python ansible usage like that in existing test cases 😞
But seems like the ansible operation is embedded,
I will try explore the python ansible api to see if there is a way to implement it

Reply:
Remove this section since it is useless by suggestion